PinpointKit/PinpointKit/Sources/TextAnnotationView.swift

```diff
@@ -50,6 +50,14 @@ public class TextAnnotationView: AnnotationView, UITextViewDelegate {
         }
     }
     
+    public func scaleControlPoints(scale: CGFloat) {
```

Comment:
We should either put a comment in these methods mentioning they're a no-op, or, since they're not really required, we could make them optional to implement on the protocol by providing no-op default implementations.